test/cql-pytest/test_tools.py

```diff
@@ -586,6 +586,15 @@ def check(self, scylla_path, extra_args, sstable, dump_reference, cwd=None, env=
         dump = list(dump.values())[0]
         assert dump == dump_reference
 
+    def check_fail(self, scylla_path, extra_args, sstable, error_msg=None):
```

Comment:
nitpick: the name "check_fail" is too general. It test a very specific tool and very specific usage of it - not some general failure.

Reply:
This is a member of a class, which provides context. Note that the above method is also just called `check()`.